feature-libs/product-configurator/rulebased/components/tab-bar/configurator-tab-bar.component.ts

```diff
@@ -171,6 +200,19 @@ export class ConfiguratorTabBarComponent {
   }
 
   /**
+   * Returns the tabindex for the configuration tab.
+   * The configuration tab is excluded from the tab chain if currently the overview page is displayed.
+   * @param pageType - Page type
```

Comment:
The same here:
Please add an empty line between an explanation and a  return value.